src/app/services/wizard-github.service.ts

```diff
@@ -0,0 +1,168 @@
+/*
+ *  Digital Excellence Copyright (C) 2020 Brend Smits
+ *
+ *   This program is free software: you can redistribute it and/or modify
+ *   it under the terms of the GNU Lesser General Public License as published
+ *   by the Free Software Foundation version 3 of the License.
+ *
+ *   This program is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty
+ *   of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *   See the GNU Lesser General Public License for more details.
+ *
+ *   You can find a copy of the GNU Lesser General Public License
+ *   along with this program, in the LICENSE.md file in the root project directory.
+ *   If not, see https://www.gnu.org/licenses/lgpl-3.0.txt
+ */
+import { HttpClient } from '@angular/common/http';
+import { GenericWizard } from './interfaces/generic-wizard';
+import { Injectable } from '@angular/core';
+import { Observable, forkJoin, of, throwError } from 'rxjs';
+import { map, switchMap, catchError } from 'rxjs/operators';
+import { GitHubContributor } from '../models/resources/external/github/contributor';
+import { MappedCollaborator } from '../models/internal/mapped-collaborator';
+import { Collaborator } from '../models/domain/collaborator';
+import { GitHubRepo } from '../models/resources/external/github/repo';
+import { MappedProject } from 'src/app/models/internal/mapped-project';
+import { StringUtils } from '../utils/string.utils';
+
+/**
+ * Service to fetch a repo and it's details from Github.
+ */
+@Injectable({
+  providedIn: 'root'
+})
+export class WizardGithubService implements GenericWizard {
```

Comment:
I thought the backend was going to handle all of the requests to the endpoints of various services because of CORS?

Reply:
Nope, the GitHub API is completely open and has no issues with CORS